src/app/clusters/wifi_network_diagnostics_server/wifi_network_diagnostics_server.cpp

```diff
@@ -67,6 +70,23 @@ CHIP_ERROR WiFiDiagosticsAttrAccess::ReadIfSupported(CHIP_ERROR (ConnectivityMan
     return aEncoder.Encode(data);
 }
 
+CHIP_ERROR WiFiDiagosticsAttrAccess::ReadWiFiBssId(AttributeValueEncoder & aEncoder)
+{
+    ByteSpan bssid;
```

Comment:
Please replace this (and others) with `Clusters::WiFiNetworkDiagnostics::Attributes::Bssid::TypeInfo::Type` so that you're in-fact, always aligned with what the XML definition is stating. This ensures that if that ever changes, you will get the correct compilation errors (as opposed to discovering those issues at run-time).